docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Pauli terms and weights hasn't been defined in this tutorial. Do we want to link to another tutorial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>